src/components/divider.tsx

```diff
@@ -8,15 +8,21 @@ export interface DividerProps {
     direction?: DividerDirection;
     /** If true a margin will be applied to the divider */
     margin?: boolean;
+    /** Width of the divider line, including the unit */
+    width?: string;
 }
 
-export const Divider: React.StatelessComponent<DividerProps> = ({ direction, margin }) => (
-    <div className={classNames(`divider divider--${direction}`, { 'divider--with-margin': margin })} />
+export const Divider: React.StatelessComponent<DividerProps> = ({ direction, margin, width }) => (
+    <div
+        className={classNames(`divider divider--${direction}`, { 'divider--with-margin': margin })}
+        style={direction === 'horizontal' ? { height: width, borderWidth: width } : { width, borderWidth: width }}
+    />
```

Comment:
This change is not covered in the test, which made the branch coverage drop to 50.